src/containers/Transactions/Simple/NFTokenCreateOffer.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+
+import Account from '../../shared/components/Account'
+import { Amount } from '../../shared/components/Amount'
+import { SimpleRow } from '../../shared/components/Transaction/SimpleRow'
+
+export interface Props {
+  data: {
+    instructions: {
+      account: string
+      amount: { currency: string; amount: number; issuer: string }
+      tokenID: string
+      isSellOffer: boolean
+      owner: string
+      offerID: string
+    }
+  }
+}
+
+const NFTokenCreateOffer = (props: Props) => {
+  const {
+    data: {
+      instructions: { offerID, account, amount, tokenID, isSellOffer, owner },
+    },
+  } = props
+
+  return (
+    <>
+      <SimpleRow label="Offer ID">
+        <div className="dt" data-test="offer-id">
+          {offerID}
+        </div>
+      </SimpleRow>
+      <div data-test="buyer-or-seller">
+        <SimpleRow label={isSellOffer ? 'Seller' : 'Buyer'}>
+          <div className="account" data-test="buyer-or-seller-account">
+            <Account account={account} />
+          </div>
+        </SimpleRow>
+      </div>
+      {!isSellOffer && (
+        <SimpleRow label="Owner">
+          <div className="account" data-test="owner">
+            <Account account={owner} />
+          </div>
+        </SimpleRow>
+      )}
+      <SimpleRow label="Token ID">
+        <div data-test="token-id">
+          <div className="dt">{tokenID}</div>
+        </div>
+      </SimpleRow>
+      <SimpleRow label="Amount">
+        <div data-test="amount">
+          <Amount
+            value={{
+              issuer: amount.issuer,
+              currency: amount.currency,
+              amount: amount.amount.toString(),
```

Comment:
As part of #270 the need to pull these out individually will be fixed.  You will then be able to pass in `amount` to `value`.  I will pull out that fix into its own PR.

Reply:
Created PR #303.